tutorials/users/ide/kernels-and-dependency-management.md

```diff
@@ -1 +1,85 @@
-# kernels-and-dependency-management
+# Kernels and Dependency Management
+
+## What is a Kernel?
+
+Kernels in Jupyter are self-contained, program language specific processes that run independently.  
+Having kernels configured and packaged for specific purposes gives the Jupyter environment some flexibility.
+A kernel doesn't need to be configured to serve every purpose imaginable. Several kernels can be made, with none
+of the dependencies for one kernel affecting the dependencies for another. If a kernel needs to be updated, changes can 
```

Comment:
Let's not talk about updating, changing, deleting kernels in this section. That is more for node admins or people who want to develop for the Marble software itself

Reply:
Removed the sentences about updating, changing, and deleting kernels.